templates/microApp/package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "name": "microApp",
```

Comment:
This name Property should be dynamic, it should be snake-cased project name user entered on command line.